expression/builtin_compare_vec_test.go

```diff
@@ -34,6 +35,7 @@ var vecBuiltinCompareCases = map[string][]vecExprBenchCase{
 			gener{defaultGener{eType: types.ETDecimal, nullRation: 0.2}},
 			gener{defaultGener{eType: types.ETDecimal, nullRation: 0.2}},
 		}},
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETInt, types.ETInt}, childrenFieldTypes: []*types.FieldType{{Tp: mysql.TypeInt24, Flag: mysql.UnsignedFlag}}},
```

Comment:
We should cover all the signed/unsigned cases of the two arguments.
You can add more `vecExprBenchCase` here.

Reply:
done.